### PR TITLE
Assert Graphviz import

### PR DESCRIPTION
**Pull request by kulbachcedric**
Hi,

the Hoeffding trees require [Graphviz](https://graphviz.org/download/) to be visualised. Therefore, I would suggest to assert whether it can be imported or not.

Best

### CODE REVIEW

**Comment by MaxHalford** [comment]
Hey hey.

It's [more Pythonic](https://realpython.com/python-lbyl-vs-eafp/) to let the exception raise itself. Here at line 430, an exception will be raised if graphviz is not installed. Did that not happen to you?

**Comment by MaxHalford** [comment]
Well we do want to keep the graphviz package optional and not bother our users. If you don't find the import error explicit enough, we can do this in the `draw` method:

```py
try:
    import graphviz
except ImportError:
    raise ValueError("You have to install graphviz to use the draw method") from e
```

File: river/tree/hoeffding_tree.py [hunk 1]
```diff
@@ -405,6 +400,10 @@ def draw(self, max_depth: int = None):
         .. image:: ../../docs/img/dtree_draw.svg
             :align: center
         """
+        try:
+            import graphviz
+        except ImportError as e:
+            raise ValueError("You have to install graphviz to use the draw method,") from e
```
Comment (MaxHalford) [inline river/tree/hoeffding_tree.py:406]: Is the comma `,` necessary?

Comment (kulbachcedric) [reply river/tree/hoeffding_tree.py:406]: Should be a `.` 🙈

**Pull request merged**